tests/trainer/test_dataloaders.py

```diff
@@ -1066,6 +1066,41 @@ def test_dataloaders_load_every_epoch(tmpdir):
         assert call['name'] == expected
 
 
+@mock.patch.dict(os.environ, {"PL_DEV_DEBUG": "1"})
+def test_dataloaders_load_every_n_epochs(tmpdir):
+
+    model = EvalModelTemplate()
+
+    # logger file to get meta
```

Comment:
What do you mean?

Reply:
hmm, I just used what was previously implemented for `test_dataloaders_load_every_epoch` for  `test_dataloaders_load_every_n_epochs`

Reply:
seems to be present in all the `test_dataloaders_`s